src/components/footer/Footer.jsx

```diff
@@ -43,6 +46,8 @@ const Footer = () => {
         </p>
       </div>
     </section>
+
+ 
```

Comment:
Useless empty lines + I'm pretty sure Prettier didn't validate this, what happened ?